internal/session/jobs.go

```diff
@@ -20,5 +24,17 @@ func RegisterJobs(ctx context.Context, scheduler *scheduler.Scheduler, w db.Writ
 		return fmt.Errorf("error registering session cleanup job: %w", err)
 	}
 
+	repo, err := NewRepository(r, w, k)
+	if err != nil {
+		return fmt.Errorf("error registering delete terminated session job: %w", err)
```

Comment:
This error message looks wrong

Reply:
It's saying that it failed to register the job, but in this context what actually failed is the creation of the new repository. The caller of this might say that registration failed, but it didn't fail on this level.